lib/editor/util/__tests__/__snapshots__/map.js.snap

```diff
@@ -0,0 +1,1244 @@
+        -77.922707,
+        42.09391,
+      ],
+      Array [
+        -77.92274499999999,
+        42.094158,
+      ],
+      Array [
+        -77.922799,
+        42.094528000000004,
+      ],
+      Array [
+        -77.922974,
+        42.095657,
+      ],
+      Array [
+        -77.923104,
+        42.096481,
+      ],
+    ],
+  ],
+  "dragId": undefined,
+  "updatedControlPoints": Array [
+    Object {
+      "defaultDwellTime": 0,
+      "defaultTravelTime": 120,
+      "distance": 0,
+      "hidden": true,
+      "id": "fyz2",
+      "permanent": true,
+      "point": Object {
+        "geometry": Object {
+          "coordinates": Array [
+            -77.92252574804179,
+            42.09235127309673,
+          ],
+          "type": "Point",
+        },
+        "properties": Object {},
+        "type": "Feature",
+      },
+      "shapeDistTraveled": 0,
+      "shapePtSequence": NaN,
+      "stopId": "a9c9cd55-ca3f-4ce7-9a93-96f2f4eaf089",
+      "timepoint": null,
+    },
+    Object {
+      "distance": 213.4093278296342,
+      "id": "0cj3",
+      "permanent": true,
+      "point": Object {
+        "geometry": Object {
+          "coordinates": Array [
+            -77.92276015989871,
+            42.09426187381858,
+          ],
+          "type": "Point",
+        },
+        "properties": Object {},
+        "type": "Feature",
+      },
+      "shapeDistTraveled": NaN,
+      "shapePtSequence": NaN,
+    },
+    Object {
+      "defaultDwellTime": 0,
+      "defaultTravelTime": 240,
+      "distance": 426.81865565926853,
+      "hidden": true,
+      "id": "1gwo",
+      "permanent": true,
+      "point": Object {
+        "geometry": Object {
+          "coordinates": Array [
+            -77.92305462145765,
+            42.096168018547736,
+          ],
+          "type": "Point",
+        },
+        "properties": Object {},
+        "type": "Feature",
+      },
+      "shapeDistTraveled": 426.81865565926853,
+      "shapePtSequence": NaN,
+      "stopId": "5b6afe4d-f4f2-43f3-810e-91749420af53",
+      "timepoint": null,
+    },
+  ],
+}
+`;
+
+exports[`editor > util > map > recalculateShape > deleting > should delete the last stop 1`] = `
+Object {
+  "coordinates": Array [
+    Array [
+      Array [
+        -77.922203,
+        42.086407,
+      ],
+      Array [
+        -77.922196,
+        42.086498,
+      ],
+      Array [
+        -77.922173,
+        42.086498,
+      ],
+      Array [
+        -77.92183700000001,
+        42.086513000000004,
+      ],
+    ],
+    Array [
+      Array [
+        -77.92183700000001,
+        42.086513000000004,
+      ],
+      Array [
+        -77.92188300000001,
+        42.086956,
+      ],
+      Array [
+        -77.921929,
+        42.087368,
+      ],
+      Array [
+        -77.922043,
+        42.088386,
+      ],
+      Array [
+        -77.922112,
+        42.088844,
+      ],
+      Array [
+        -77.922135,
+        42.089099,
+      ],
+    ],
```

Comment:
The coordinates in this array don't go all the way to the new last stop. Is this the desired behavior?

This is what the coordinates look like before deleting the last stop:

<img width="332" alt="Screen Shot 2019-04-02 at 5 28 57 PM" src="https://user-images.githubusercontent.com/3112493/55444770-d6cb7e00-556c-11e9-854d-1f6212dc9229.png">

And here are those coordinates after:

<img width="362" alt="Screen Shot 2019-04-02 at 5 29 33 PM" src="https://user-images.githubusercontent.com/3112493/55444791-ecd93e80-556c-11e9-8af0-a4244db2f900.png">